src/java.base/share/classes/sun/security/rsa/RSAPublicKeyImpl.java

```diff
@@ -76,10 +76,6 @@ public final class RSAPublicKeyImpl extends X509Key implements RSAPublicKey {
 
     private static NativeCrypto nativeCrypto;
 
-    static {
-        nativeCrypto = nativeCrypto.getNativeCrypto();
```

Comment:
There's a typo in the old code `nativeCrypto.getNativeCrypto()` should have been `NativeCrypto.getNativeCrypto()` - I'm surprised it even compiled as the field `nativeCrypto` has not been initialized yet.

Reply:
Just a compilation warning
```
openj9-openjdk-jdk/src/java.base/share/classes/sun/security/rsa/RSAPublicKeyImpl.java:80: warning: [static] static method should be qualified by type name, NativeCrypto, instead of by an expression
        nativeCrypto = nativeCrypto.getNativeCrypto();
```